dts/bindings/serial/st,stm32-uart.yaml

```diff
@@ -18,7 +18,7 @@ properties:
         Enable the single wire half-duplex communication.
         Using this mode, TX and RX lines are internally connected and
         only TX pin is used afterwards and should be configured.
-        RX/TX conflicts should be handled on user side.
+        RX/TX conflicts must be handled on user side.
```

Comment:
Please move this to previous commit.

Reply:
oh, yeah messed something up there. Done.